<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoraraWidget</class>
 <widget class="QWidget" name="SoraraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>611</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>611</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/bkg2);
color: rgb(0, 170, 255);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_p2pweb">
        <attribute name="title">
         <string>SORARA web</string>
        </attribute>
        <widget class="QTextEdit" name="contentsTextEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>981</width>
           <height>391</height>
          </rect>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="urlPlainTextEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>971</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="reloadPushButton">
         <property name="geometry">
          <rect>
           <x>850</x>
           <y>440</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>reload</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_p2pmessage">
        <attribute name="title">
         <string>SORARA message</string>
        </attribute>
        <widget class="QTextEdit" name="messagesTextEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>971</width>
           <height>441</height>
          </rect>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="messagePlainTextEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>450</y>
           <width>881</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="messagesendPushButton">
         <property name="geometry">
          <rect>
           <x>890</x>
           <y>450</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
